plugins/service-monitor-cloudera/service-monitor-cloudera-service/src/main/java/com/thinkbiganalytics/servicemonitor/rest/client/cdh/ClouderaClient.java

```diff
@@ -75,10 +76,12 @@ public void setClouderaManagerClientBuilder() {
         Integer port = new Integer(portString);
         String username = clientConfig.getUsername();
         String password = clientConfig.getPassword();
-        LOG.info("Created New Cloudera Client for Host [" + host + ":" + port + "], user: [" + username + "]");
-        this.clouderaManagerClientBuilder =
-            new ClouderaManagerClientBuilder().withHost(host).withPort(port).withUsernamePassword(username, password);
 
+        LOG.info("Created New Cloudera Client for Host [" + host + ":" + port + "], user: [" + username + "]");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Use the built-in formatting to construct this argument. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2629)